apps/condo/pages/_app.tsx

```diff
@@ -114,6 +115,7 @@ const MenuItems: React.FC = () => {
     const { obj: billingCtx } = BillingContext.useObject({ where: { integration: { id: sppBillingId }, organization: { id: orgId } } })
     const anyReceiptsLoaded = Boolean(get(billingCtx, 'lastReport', null))
     const hasAccessToBilling = (get(role, 'canReadPayments', false) || get(role, 'canReadBillingReceipts', false)) && !isAssignedVisibilityType
+    const canManageNewsItems = get<boolean>(role, 'canManageNewsItems', false)
```

Comment:
Isn't the type inherited from the function's last argument?
```suggestion
    const canManageNewsItems = get(role, 'canManageNewsItems', false)
```

Reply:
Unfortunately no
<img width="869" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/b12f8456-ea6e-42d3-982b-e7303a4b41d2">